ci/build-unix.yml

```diff
@@ -22,7 +22,9 @@ steps:
 
   - bash: ./fmt.sh --test
     displayName: 'Platform-agnostic lints and checks'
-    condition: and(succeeded(), eq(variables['Agent.OS'], 'Linux'))
+    # Temporary hack for 0.13.55. Should be removed once published.
+    condition: and(succeeded(), eq(variables['Agent.OS'], 'Linux'),
+                   ne(${{parameters.is_release}}, 'true'))
```

Comment:
In the example at https://docs.microsoft.com/en-us/azure/devops/pipelines/process/conditions?view=azure-devops&tabs=yaml#use-a-template-parameter-as-part-of-a-condition the parameter is quoted. I guess that doesn’t matter?

Reply:
It's a trap! See #4745.

Reply:
:man_facepalming: